userspace/libscap/engine/modern_bpf/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+include_directories(${LIBSCAP_INCLUDE_DIRS})
+
+# Option to build bpf tests
+option(BUILD_MODERN_BPF_TEST "Build tests for modern BPF" OFF)
```

Comment:
I understand why this is OFF by default, but we need a new test-infra job to test the build for this or put the default to ON for now.  
Moreover, it is only used by scap-open example at the moment, therefore there should be no issue defaulting to ON (and later, we would have the vtable `match` method to understand which driver to be used).

Actually, i think i'd even drop this flag entirely, but i understand we'd forcefully add a dependency in that case.

Reply:
ah sorry, you are looking at the option for tests, yeah I agree with you it can become `ON` by default. Right now there is no test framework so I will enable it by default when I will add tests